web/cobrands/northamptonshire/base.scss

```diff
@@ -13,15 +13,21 @@
 
 #site-logo {
     width: 190px;
-    height: 41px;
-    padding: 11px 0;
-    background: url(/cobrands/northamptonshire/img/new-ncc-logo.png) 0 50% no-repeat;
+    // height: 41px;
+    // padding: 11px 0;
```

Comment:
Any reason to keep these commented-out lines?

Reply:
Good spot, I'll remove them. Thanks